Code/Tools/SceneAPI/SceneCore/DataTypes/Rules/IMeshAdvancedRule.h

```diff
@@ -11,14 +11,16 @@
 #include <AzCore/std/string/string.h>
 #include <AzCore/RTTI/RTTI.h>
 #include <SceneAPI/SceneCore/DataTypes/Rules/IRule.h>
+#include <AzCore/std/string/fixed_string.h>
 
 namespace AZ
 {
     namespace SceneAPI
     {
         namespace DataTypes
         {
-            const static AZStd::string s_advancedDisabledString = "Disabled";
+            static AZStd::string s_advancedDisabledString = "Disabled";
```

Comment:
I believe you meant to change this to a fixed_string.
But I think you are better off changing this to a `const char*` and getting rid of the static since it is declared in header file.
There shouldn't be multiple copies of this literal per translation unit.

Reply:
I have to do static because is being included in multiple files which end up in different translation units

Reply:
You can use `inline` for that.